### DISCUSSION TITLE
Modded Deepslate Ores

### DISCUSSION THREAD

**Discussion by Redgi110** [Roadhog360/Et-Futurum-Requiem#359] [Ideas]
Title says everything. Deepslate variants for ores from mods.

**Comment by Roadhog360** [comment]
Already planned

**Comment by GregoriusT** [comment]
So if we use a 16 metadata ore block, which 16 ores would make most sense to add to it? because spamming single metadata ore blocks will cause issues by using up item and block ids for no reason.

One way would be using the Ore Blocks you already have and have a different metadata on each of those represent a different mod ore. You got 8 ids used up by your deepslate ores, this makes 8 * (16 - 1) = 120 Mod Ores you can add by just adding to the metadata of your existing Ore Blocks.

Ores that are a must to include would be:

Tin, Zinc, Lead (meta of Copper Ore)
Nickel, Cobalt, Rutile/Titanium, Tungsten (meta of Iron Ore)
Aluminium, Silver, Platinum (meta of Gold Ore)
Ruby, Sapphire, GreenSapphire (meta of Diamond Ore, because Sapphire is second hardest compared to Diamond)
Aquamarine (meta of Emerald Ore, because it is a type of Emerald)
Salt, Amber, Cinnabar, Sulfur, Saltpeter, Niter, CertusQuartz, ChargedCertusQuartz (meta of Lapis)
Uranium, Thorium, Draconium, Nikolite, Teslatite (meta of Redstone, since it could have the glow effect for example)

**Reply by Roadhog360** [reply]
Some of these ores have special properties and will probably need to be separate blocks to replicate that.
for "basic" ores I'll probably just add 1 ID per 16 ores.

**Reply by GregoriusT** [reply]
why  waste so many block ids for nothing though?

**Reply by Roadhog360** [reply]
I can only put 16 versions of a block under 1 id though??? How is putting 16 basic ores in 1 ID a "waste"?

**Reply by GregoriusT** [reply]
your normal ores are all 1 ID 1 Meta, adding one more ID with 16 metas is wasting one ID you could save by using the remaining 15 metas of the old deepslate ore blocks. ;)

**Reply by Roadhog360** [reply]
I just wasn't thinking, I didn't need to do that I agree it just happened because the vanilla ores were made like that too, but I will use 1 new ID for the new ores because certus quartz is not lapis ore so it shouldn't go under the lapis ore ID. It'll be under the `modded_deepslate_ore` ID

**Reply by GregoriusT** [reply]
oh right charged certus wouldve fit more into the redstone block since it glows anyways. Ive put it in lapis because its a gem lol

**Reply by Roadhog360** [reply]
If the ore has special properties or a special renderer it'll need to be a separate ID though, I think it uses an emissive texture thus probably uses a custom renderer, thus likely needs a different ID.
